templates/projects/task_presenter_editor.html

```diff
@@ -329,6 +329,8 @@ <h4 class="modal-title"><strong>{{project.name}}</strong>: {{ _('Task Presenter
         <script type="text/javascript" src="{{ ASSET_URL }}"></script>
     {% endassets %}
 
+    <script type="text/javascript" src="{{url_for('static', filename='js/task-presenter-tabs.js')}}"></script>
```

Comment:
Not sure where the theme build checks are failing, but Xi suggests we add a Jira item for fixing the checks so that we don't forget and keep making changes and ignoring the failed builds.